projects/js-packages/publicize-components/src/social-store/actions/social-notes-settings.js

```diff
@@ -30,6 +30,28 @@ export function* updateSocialNotesSettings( settings ) {
 	}
 }
 
+/**
+ * Yield actions to update settings
+ *
+ * @param {object} config - config to update
+ * @yields {object} - an action object.
+ * @returns {object} - an action object.
+ */
+export function* updateSocialNotesConfig( config ) {
+	const prevConfig = select( SOCIAL_STORE_ID ).getSocialNotesConfig();
+	const newConfig = { ...prevConfig, ...config };
+	try {
+		yield setJetpackSettings( { social_notes_config: newConfig } );
+		yield updateJetpackSettingsControl( { social_notes_config: config } );
+		const updatedSettings = yield fetchJetpackSettings();
+		yield setJetpackSettings( updatedSettings );
+		return true;
+	} catch ( e ) {
+		yield setJetpackSettings( { social_notes_config: newConfig } );
```

Comment:
If this is happening in the catch block, should this be `{ social_notes_config: prevConfig }`?

Reply:
Oh yes, good catch. Thanks

Reply:
Fixed. Thanks